modules/phase_field/include/postprocessors/GrainTracker.h

```diff
@@ -111,6 +117,15 @@ class GrainTracker : public FeatureFloodCount, public GrainTrackerInterface
    */
   void remapGrains();
 
+  // re-merge grains due to misorientation angle from euler angles calculation
+  virtual void mergeGrainsBasedMisorientation();
+
+  const EulerAngleProvider & _euler;
+  misoriAngle_isTwining _s_misoriTwin;  
```

Comment:
This doesn't follow our naming conventions. Normally for classes and methods we use CamelCase exclusively and for variables we'll use underscores to separate words. The type here is a combination of those two conventions. Let's switch it to CamelCase.

Reply:
Based on your suggestion as well as [Naming](https://mooseframework.inl.gov/workshop/index.html#/14/4), I normalized a few names:  misoriAngle_isTwining  to MisorientationAngleData; _s_misoriTwin to _s_misorientation_angle; quatReal to QuatReal; _merge_grains_basedMisorAngle to _merge_grains_based_misorientaion;